src/utils/eval_tuner.ts

```diff
@@ -0,0 +1,224 @@
+import { getRandomItem } from '.'
+import { setRandomSeed } from 'bun:jsc'
+import type { Tensor } from '@shumai/shumai'
+
+setRandomSeed(Math.random() * 100000)
+
+export enum EvalTunerType {
+  STOCH_HILL_CLIMB
```

Comment:
if we have only one type, could we just remove this for now?
in fact, rather than enum we could just pass an eval function that adheres to the protocol of StochHillClimb

Reply:
FastOptimizer is actually a separate class, which is added to the enum. I'll update the PR w that code this evening once I've had the time to go back through one last time.

Reply:
Actually, on second thought, allowing passing in a custom eval function makes a ton of sense; I'll refactor so that can be done rather than relying on an enum.